crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -252,6 +252,12 @@ mod tests {
       transport_pool:
         - type: webrtc
           signal_url: wss://signal.holotest.net
+          webrtc_config: {
+            "iceServers": [
+              { "urls": "stun:stun.l.google.com:19302" },
+              { "urls": "stun:stun2.l.google.com:19302" }
+            ]
+          }
```

Comment:
TODO - update these to match the sbd production server to be created and holochain production stun servers to be created.